lib/jefferies_tube/capistrano/ssh.rb

```diff
@@ -2,7 +2,8 @@
 task :ssh do
   on roles(:app), primary: true do |host|
     port = host.port || 22
-    puts "ssh #{host.user}@#{host} -p #{port}"
-    exec "ssh #{host.user}@#{host} -p #{port}"
+    cmd = %Q|ssh #{host.user}@#{host} -p #{port} -t "cd /u/apps/stimmi/current && exec bash -l"|
```

Comment:
i was poking around in JT source and noticed you wrote this to only work on stimmi :)